android/build.gradle

```diff
@@ -80,7 +80,7 @@ buildscript {
 
     ext.versions = [
         'jdk': 17,
-        'minSdk': 21,
+        'minSdk': 26,
```

Comment:
This would impact all Filament users on Android, esp. our internal Google ones. If it's because you are calling API 26 specific APIs, they should be guarded with API level checks instead.

Reply:
We need API26 for AHardwareBuffer. I am not sure what the best way to proceed is, but the min version is not incorrect anymore for the Filament backend.

Reply:
See example here to see how to use an API wrt target Android version: https://github.com/google/filament/blob/main/filament/backend/src/opengl/platforms/PlatformEGLAndroid.cpp#L282